internal/features/features.go

```diff
@@ -55,6 +58,10 @@ var features = map[string]bool{
 	// opt-out from v0.28
 	GitAllBranchReferences: true,
 
+	// GitSparseCheckout
+	// opt-out from v0.42
+	GitSparseCheckout: true,
```

Comment:
Let's make this opt-in for now. We can switch this to opt-out once sparse checkout is no longer an experimental feature in Git.